packages/nodejs/src/client.ts

```diff
@@ -159,6 +170,52 @@ export class BaseClient implements Client {
     return demo(this.tracer())
   }
 
+  /**
+   * Sets up the AppSignal logger with the output based on the `log` config option. If
+   * the log file is not accessible, stdout will be the output.
+   */
+  private setUpLogger(): winston.Logger {
+    const logFilePath = this.config.logFilePath
+    const logLevel = String(this.config.data["logLevel"])
+    let transport
+
+    const logFormat = printf(({ level, message, timestamp }) => {
+      return `[${timestamp} (process) #${process.pid}][appsignal][${level}] ${message}`
+    })
+
+    if ((this.config.data["log"] = "file" && logFilePath)) {
+      transport = new winston.transports.File({ filename: logFilePath })
+    } else {
+      transport = new winston.transports.Console()
+    }
+
+    return winston.createLogger({
+      format: combine(timestamp(), logFormat),
+      level: this.translateLogLevel(logLevel),
+      transports: [transport]
+    })
+  }
+
+  /**
+   * Translates our logLevel to the one supported by Winston
+   */
+  private translateLogLevel(level: string): string {
+    switch (level) {
+      case "error":
+        return "error"
+      case "warning":
+        return "warn"
+      case "info":
+        return "info"
+      case "debug":
+        return "debug"
+      case "trace":
+        return "silly"
```

Comment:
sorry what? 😅

Reply:
Yup, 😄 . Apparently, it is: "log a silly amount of information" and it's similar to "trace".  

https://github.com/winstonjs/winston#logging-levels